ui/app/src/components/SiteSearchPathSelector/PathSelector.tsx

```diff
@@ -138,28 +182,55 @@ export function PathSelector(props: PathSelectorProps) {
     document.addEventListener(idCancel, cancelCallback, false);
   };
 
+  const handleBasePathChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    onPathSelected((event.target as HTMLInputElement).value);
+  };
+
   return (
-    <Paper
-      variant="outlined"
-      onClick={disabled ? null : onOpenPathSelectionDialog}
-      className={cx(classes.pathSelectorWrapper, disabled && 'disabled')}
-    >
-      <InputBase
-        classes={{ root: classes.pathSelectorInputRoot, input: classes.invisibleInput }}
-        disabled={disabled}
-        readOnly
-        value={path}
-        placeholder={formatMessage(messages.searchIn)}
-        startAdornment={<SearchIcon className={classes.pathSelectorSearchIcon} />}
-        endAdornment={
-          !disabled && value ? (
-            <IconButton onClick={onClean} size="small">
-              <CloseIcon />
-            </IconButton>
-          ) : null
-        }
-      />
-    </Paper>
+    <>
+      <FormControl className={classes.basePathSelectorContainer}>
+        <RadioGroup value={radioBasePath} onChange={handleBasePathChange}>
```

Comment:
The value of the base path radio should still be checked when a subpath of his is selected. This won't work.